### ISSUE TITLE
[Refactor] Replace Epidata calls with `delphi-epidata-py`

### ISSUE THREAD

**Issue by dshemetov** [cmu-delphi/delphi-epidata#942]
Pretty low priority, but could be a good first issue to familiarize with this codebase and learn to use the new epidata package.

Depends on the [release](https://github.com/cmu-delphi/delphi-epidata-py/issues/11) of `delphi-epidata-py`, which is currently blocked by personnel resources.
